src/lib/annotations/doc/docAnnotatorUtil.js

```diff
@@ -359,47 +359,34 @@ export function isValidSelection(selection) {
 /**
  * Gets the context an annotation should be drawn on.
  *
- * @param {HTMLElement} pageEl - The DOM element for the current page
- * @param {string} annotationLayerClass - The class name for the annotation layer
- * @param {number} [paddingTop] - The top padding of each page element
- * @param {number} [paddingBottom] - The bottom padding of each page element
+ * @param {HTMLElement} pageEl The DOM element for the current page
+ * @param {string} annotationLayerClass The class name for the annotation layer
+ * @param {number} [paddingTop] The top padding of each page element
+ * @param {number} [paddingBottom] The bottom padding of each page element
  * @return {RenderingContext|null} Context or null if no page element was given
  */
 export function getContext(pageEl, annotationLayerClass, paddingTop, paddingBottom) {
     if (!pageEl) {
         return null;
     }
 
-    let annotationLayerEl = pageEl.querySelector(`.${annotationLayerClass}`);
-    let context;
     // Create annotation layer if one does not exist (e.g. first load or page resize)
+    let annotationLayerEl = pageEl.querySelector(`.${annotationLayerClass}`);
     if (!annotationLayerEl) {
         annotationLayerEl = document.createElement('canvas');
         annotationLayerEl.classList.add(annotationLayerClass);
+
         const pageDimensions = pageEl.getBoundingClientRect();
         const pagePaddingTop = paddingTop || 0;
         const pagePaddingBottom = paddingBottom || 0;
-        const pxRatio = window.devicePixelRatio || 1;
-        const width = pageDimensions.width;
-        const height = pageDimensions.height - pagePaddingTop - pagePaddingBottom;
-
-        annotationLayerEl.width = pxRatio * width;
-        annotationLayerEl.height = pxRatio * height;
-        context = annotationLayerEl.getContext('2d');
-
-        if (pxRatio !== 1) {
-            annotationLayerEl.style.width = `${width}px`;
-            annotationLayerEl.style.height = `${height}px`;
-            context.scale(pxRatio, pxRatio);
-        }
+        annotationLayerEl.width = pageDimensions.width;
+        annotationLayerEl.height = pageDimensions.height - pagePaddingTop - pagePaddingBottom;
```

Comment:
Move this to below the `if` so that the canvas is resized on every call of `getContext()`, like we discussed yesterday. If you want bonus points, I'd love it if you moved it out entirely from this function and have one that's job is to resize the canvas.

ie)
```
// clear and resize
resizeContext() {
  canvas.width = blah;
  canvas.height = blargh;
}
```